src/client/replication_ddl_client.cpp

```diff
@@ -1217,23 +1219,34 @@ replication_ddl_client::query_backup_policy(const std::vector<std::string> &poli
     configuration_query_backup_policy_response resp;
     ::dsn::unmarshall(resp_task->get_response(), resp);
 
+    std::streambuf *buf;
+    std::ofstream of;
+    buf = std::cout.rdbuf();
+    std::ostream out(buf);
+
     if (resp.err != ERR_OK) {
         return resp.err;
     } else {
+        dsn::utils::multi_table_printer mtp;
         for (int32_t idx = 0; idx < resp.policys.size(); idx++) {
             if (idx != 0) {
                 std::cout << "************************" << std::endl;
             }
             const policy_entry &pentry = resp.policys[idx];
-            std::cout << "policy_info:" << std::endl;
-            print_policy_entry(pentry);
-            std::cout << std::endl << "backup_infos:" << std::endl;
+            dsn::utils::table_printer tp_policy = print_policy_entry(pentry);
+            mtp.add(std::move(tp_policy));
             const std::vector<backup_entry> &backup_infos = resp.backup_infos[idx];
+            dsn::utils::table_printer tp_backup("backup_infos");
```

Comment:
As cold_backup_constant::BACKUP_INFO is considered as the reserve key [1], so use cold_backup_constant::BACKUP_INFO instead of "backup_infos".

1. https://github.com/apache/incubator-pegasus/blob/master/src/meta/meta_backup_service.cpp#L1470